lib/Calendar.js

```diff
@@ -89,7 +89,9 @@ function createCalendar(request) {
 
             const xmlRequest = byTimeTemplate({ start, end });
 
-            return request(this.config, "REPORT", 1, xmlRequest).then(xmlParser.parseEvents);
+            return request(this.config, "REPORT", 1, xmlRequest)
+                .then(xmlParser.parseEvents)
+                .then(events => events.filter(event => moment(event.data.start).isSameOrAfter(start, "day")));
```

Comment:
If I remember correctly you told me that returning `recurring event occurrences from the past` is part of the caldav spec, isn't it? 
So in that case you shouldn't filter past events by default, but you should expose some kind of a static helper function that could be easily used along with scrapgoat.